pkg/process/procutil/process_linux.go

```diff
@@ -123,6 +160,110 @@ func (p *Probe) getCmdline(pidPath string) []string {
 	return trimAndSplitBytes(cmdline)
 }
 
+// parseStatus retrieves status info from "status" file for a process in procfs
+func (p *Probe) parseStatus(pidPath string) *statusInfo {
+	path := filepath.Join(pidPath, "status")
+	var err error
+
+	sInfo := &statusInfo{
+		uids:        make([]int32, 0),
```

Comment:
Since we're allocating `uids` and `gids`  inside `parseStatusKV`, can we only use `[]int32{}`, here? Or there's no penalty in making a call to `make`?

Reply:
I don't have an opinion, just dropping this here: https://stackoverflow.com/questions/29164375/correct-way-to-initialize-empty-slice

Reply:
I think it's idiom to do `[]int32{}`, because that doesn't allocate any memory, whereas `make` would allocate a 0 size slice. I'll change them to `[]int32{}` but in reality it probably doesn't make much difference.